docs/upgrade.html

```diff
@@ -74,24 +74,24 @@ <h5><a id="upgrade_400_notable" href="#upgrade_400_notable">Notable changes in 4
         <li>
             The Next Generation of the Consumer Rebalance Protocol (<a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-848%3A+The+Next+Generation+of+the+Consumer+Rebalance+Protocol">KIP-848</a>)
             is now Generally Available (GA) in Apache Kafka 4.0. The protocol is automatically enabled on the server when the upgrade to 4.0 is finalized.
-            Note that once the new protocol is used by consumer groups, the cluster can only downgrade to version 3.4.1 or newer.
-            Check <a href="/{{version}}/documentation.html#consumer_rebalance_protocol">here</a> for details.
+            Note that once the new protocol is used by consumer groups, the cluster can only be downgraded to version 3.4.1 or newer.
+            For more information check <a href="/{{version}}/documentation.html#consumer_rebalance_protocol">here</a>.
         </li>
         <li>
-            Transactions Server Side Defense (<a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-890%3A+Transactions+Server-Side+Defense">KIP-890</a>)
+            Transactions Server-Side Defense (<a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-890%3A+Transactions+Server-Side+Defense">KIP-890</a>)
```

Comment:
ditto: Could you also update the KIP link to `https://cwiki.apache.org/confluence/x/B40ODg`? If the KIP title changes in the future, the current URL may become invalid.